plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/editors/data/preferences/PrefPageResultSetPresentation.java

```diff
@@ -105,7 +112,13 @@ protected void loadPreferences(DBPPreferenceStore store)
         try {
         	autoSwitchMode.setSelection(store.getBoolean(ResultSetPreferences.RESULT_SET_AUTO_SWITCH_MODE));
             showDescription.setSelection(store.getBoolean(ResultSetPreferences.RESULT_SET_SHOW_DESCRIPTION));
-            columnWidthByValue.setSelection(store.getBoolean(ResultSetPreferences.RESULT_SET_CALC_COLUMN_WIDTH_BY_VALUES));
+            ResultSetPreferences.GridColumnCalcWidthMethod calcColumnWidthMethod =
+                CommonUtils.valueOf(ResultSetPreferences.GridColumnCalcWidthMethod.class,
+                store.getString(ResultSetPreferences.RESULT_SET_CALC_COLUMN_WIDTH_METHOD));
+            columnWidthCalcCombo.select(
+                calcColumnWidthMethod == ResultSetPreferences.GridColumnCalcWidthMethod.VALUES ? 1 :
+                calcColumnWidthMethod == ResultSetPreferences.GridColumnCalcWidthMethod.TITLE_DESCRIPTION_VALUES ?
+                    2 : 0);
```

Comment:
If the combo's values' indices match the indices of the values of the enum, you can use `.ordinal()` to retrieve its index